src/gmp.c

```diff
@@ -6273,16 +6273,6 @@ gmp_xml_handle_start_element (/* unused */ GMarkupParseContext* context,
                                        attribute_names,
                                        attribute_values);
 
-            /* Special case details with default 1, for backward
-             * compatibility. */
-            if (find_attribute (attribute_names, attribute_values,
-                                "details", &attribute))
-              get_reports_data->report_get.details = strcmp (attribute, "0");
-            else
-              get_reports_data->report_get.details = 1;
-            get_reports_data->get.details
```

Comment:
If you look a few lines up both data structs are being filled the same way,
so I don't see a need to make sure they're the same.  I think the old code
was just filling get.details from report_get.details because it was the simplest
option.

Reply:
Sorry, I was looking mostly at the diff and misremembered how the differences between the two were handled when I wrote that comment.